### ISSUE TITLE
Relationship between simple/composite tokens to a 3-tiered token structure

### ISSUE THREAD

**Issue by jkinley** [lukasoppermann/style-dictionary-utils#16]
Hi Lukas. My colleagues are working on implementing W3C tokens and noticed a pattern. The general idea is that once you start using W3C Composite tokens, then you are in the realm of semantic or component level tokens. Let me know your thoughts.

So for example, at the Core Token level you might have W3C Simple tokens for, say, border width:

```
/* core.border-width.json5 */

{
  core: {
    'border-width': {
        "$type": 'dimension',
        '0': {"$value": '1px'},
        '1': {"$value": '1px'},
        '2': {"$value": '2px'},
      },
  }
}
```

But then at the Semantic Token level, you would use W3C Composite Tokens to create something like this by referencing simple core tokens:

```
/* semantic.border.json5 */

{
  semantic: {
    border-success: {
      "$type": 'border',
      "$value": {
        'color': '{core.color.green.500}',
        'width': '{core.border-width.1}',
        'style': '{core.border-style.solid}'
      }
    }
  }
}
```
The main concept is that once you build a "complete" border it can't help but have semantic meaning or is a fully derived component level token.

**Comment by lukasoppermann** [comment]
> My point being that you are probably not going to be using W3C composite tokens at the core level.

No, probably not. But it is possible. You could create composite tokens e.g. for shadows that get reused as functional tokens. It depends very much on your setup.
